### PR TITLE
[Samples] Update some of the samples to use v1.3 features

### PR DESCRIPTION
**Pull request by almedina-ms**
## Related Issue
References #4552 

ADO 23713386
ADO 23703188
ADO 23702956
ADO 23702320

## Description
Updates some of the samples that contain Inputs to use the `label`, `errorMessage`, `isRequired`, and the validation properties to update users into our v1.3 story

Waiting on the message to update older cards

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/4561)

### CODE REVIEW

File: samples/v1.3/Scenarios/InputsWithValidation.json [hunk 1]
```diff
@@ -1 +1,199 @@
...
						"type": "Action.Submit",
						"title": "OK"
					}
				]
			}
		}
	]
}
\ No newline at end of file
+{
```
Comment (paulcam206) [inline samples/v1.3/Scenarios/InputsWithValidation.json:1]: thanks for catching this :)

File: samples/v1.3/Scenarios/ActivityUpdate.json [hunk 2]
```diff
@@ -0,0 +1,118 @@
+{
```
Comment (paulcam206) [inline samples/v1.3/Scenarios/ActivityUpdate.json:1]: We also need to update the templated version of `Scenarios` cards

Comment (almedina-ms) [reply samples/v1.3/Scenarios/ActivityUpdate.json:1]: Do we have some data to update the templated versions or do we have to create new data cards?

File: samples/v1.0/Elements/Input.ChoiceSet.json [hunk 3]
```diff
@@ -3,6 +3,18 @@
...
 	"version": "1.0",
 	"body": [
+		{
+			"type": "Container",
+			"verticalContentAlignment": "Center",
+			"style": "warning",
+			"items": [
+				{
+					"type": "TextBlock",
+					"text": "**IMPORTANT:** This card is based on an older schema and doesn't make use of the label property for its Input elements. Because of this, it doesn't properly support accessibility. It's included here for historical purposes. Please see [[NEW SAMPLE NAME]] for an updated version of this card.",
```
Comment (golddove) [inline samples/v1.0/Elements/Input.ChoiceSet.json:13]: Can we replace `[[NEW SAMPLE NAME]]` with link to new sample?

File: samples/v1.3/Scenarios/ExpenseReport.json [hunk 4]
```diff
@@ -0,0 +1,824 @@
+{
```
Comment (golddove) [inline samples/v1.3/Scenarios/ExpenseReport.json:1]: This uses the old ExpenseReport.json which has an accessibility bug. Can you update to use the latest one from main?

File: samples/v1.3/Elements/Input.Text.InlineAction.json [hunk 5]
```diff
@@ -0,0 +1,49 @@
+{
```
Comment (golddove) [inline samples/v1.3/Elements/Input.Text.InlineAction.json:1]: We decided not to publicly show the updated Element cards until most of our hosts move to v1.3, so this needs to be renamed or put in a different folder so it won't be surfaced until we are ready.

File: samples/v1.3/Scenarios/CalendarReminder.json [hunk 6]
```diff
@@ -0,0 +1,62 @@
+{
```
Comment (golddove) [inline samples/v1.3/Scenarios/CalendarReminder.json:1]: These new cards need a new name (like "Calendar Reminder with labels")

Or else the site gets very confused:
https://adaptivecardsci.z5.web.core.windows.net/pr/4561/samples/CalendarReminder.html

File: samples/v1.3/Elements/Input.Text.InlineAction.json (hunk 5 above)
Comment (golddove) [reply samples/v1.3/Elements/Input.Text.InlineAction.json:1]: Which is tracked by #4612

File: samples/v1.3/Scenarios/ActivityUpdate.json (hunk 2 above)
Comment (paulcam206) [reply samples/v1.3/Scenarios/ActivityUpdate.json:1]: I think we already have some data you can use (see [here](https://github.com/microsoft/AdaptiveCards/blob/main/samples/Templates/Scenarios/ActivityUpdate.data.json))

File: samples/v1.0/Scenarios/InputForm.json [hunk 7]
```diff
@@ -3,6 +3,18 @@
...
+			"verticalContentAlignment": "Center",
+			"style": "warning",
+			"items": [
+				{
+					"type": "TextBlock",
+					"text": "**IMPORTANT:** This card is based on an older schema and doesn't make use of the label property for its Input elements. Because of this, it doesn't properly support accessibility. It's included here for historical purposes. Please see [Input Form With Labels](https://adaptivecards.io/samples/InputFormWithLabels.html) for an updated version of this card.",
+					"wrap": true
+				}
+			]
+		}
```
Comment (paulcam206) [inline samples/v1.0/Scenarios/InputForm.json:17]: oops! dropped a `,`:

```suggestion
		},
```

**Review by paulcam206 — CHANGES_REQUESTED** [review]

File: samples/v1.0/Scenarios/InputForm.json (hunk 7 above)
Comment (paulcam206) [reply samples/v1.0/Scenarios/InputForm.json:17]: @almedina-ms - I think this is the reason why the PR builds were failing in tests :)

File: samples/v1.3/Scenarios/ActivityUpdate.json (hunk 2 above)
Comment (almedina-ms) [reply samples/v1.3/Scenarios/ActivityUpdate.json]: Fixed

**Review by paulcam206 — APPROVED** [review]

**Comment by shalinijoshi19** [comment]
@almedina-ms we should get this checked in

**Comment by golddove** [comment]
@almedina-ms the `ExpenseReportWithLabels.json` here still reflects the old ExpenseReport sample, which has accessibility issues. Please incorporate from the latest `ExpenseReport.json`

**Comment by shalinijoshi19** [comment]
@almedina-ms update here? @jwoo-msft the .NET builds continue to fail

**Comment by shalinijoshi19** [comment]
@almedina-ms is this PR still valid or does it conflict iwth #4561 ?

**Comment by shalinijoshi19** [comment]
This PR seems stale. Closing out.

**Pull request closed without merging**